src/test/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/javadocTags/expectedLeadingAsteriskAfterSeeTagAst.txt

```diff
@@ -0,0 +1,32 @@
+JAVADOC -> JAVADOC [0:0]
+|--LEADING_ASTERISK -> * [0:0]
+|--WS ->   [0:1]
+|--JAVADOC_TAG -> JAVADOC_TAG [0:2]
+|   |--SEE_LITERAL -> @see [0:2]
+|   |--NEWLINE -> \n [0:6]
+|   |--LEADING_ASTERISK -> * [1:0]
+|   |--WS ->   [1:1]
+|   |--STRING -> "string" [1:2]
```

Comment:
Why do we need `STRING` when description should fill this void, right?
Why is `STRING` not a description?

Reply:
http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see
it is `@see "string"`, description is not allowed as html tags are not allowed in `see`

Reply:
@rnveach 
As roman said, just to conform to `@see` syntax more closely

Reply:
>  description is not allowed as html tags are not allowed in see

@romani Look at current grammar definition for see, it has `description` in it.
>       | SEE_LITERAL (WS | NEWLINE | {!isNextJavadocTag()}? LEADING_ASTERISK)* reference?
>           (STRING | htmlElement)* (WS | NEWLINE)* ((WS | NEWLINE) description)?

> it is `@see "string"`

description already supports `"string"`.
See: 
````
    |  |  |  |      |--JAVADOC_TAG -> JAVADOC_TAG [2:4]
    |  |  |  |      |  |--PARAM_LITERAL -> @param [2:4]
    |  |  |  |      |  |--WS ->  [2:10]
    |  |  |  |      |  |--PARAMETER_NAME -> a [2:11]
    |  |  |  |      |  |--WS ->  [2:12]
    |  |  |  |      |  `--DESCRIPTION -> DESCRIPTION [2:13]
    |  |  |  |      |      `--TEXT -> "string"  [2:13]
````

Reply:
@rnveach Description shall be removed #4986 . Also having `STRING` in a parent like `description` hides away the grammar structure I feel. So even if we kept `description`, `STRING`s should be present separately in the javadoc tree, I feel.